acceptance-tests/helpers/apps/payload.go

```diff
@@ -0,0 +1,15 @@
+package apps
+
+import (
+	"encoding/json"
+	"reflect"
+
+	. "github.com/onsi/gomega"
+)
+
+type Payload string
+
+func (p Payload) Parse(receiver any) {
```

Comment:
suggestion: I know it's test code and all, but it seems that the name `ParseInto` would have a clearer intent statement. Otherwise, one might assume that the argument is the entity being parsed, even though the result of an HTTP call is very unlikely to be a parser.

Reply:
Agreed